src/neural/backends/network_onnx.cc

```diff
@@ -319,24 +323,39 @@ Ort::SessionOptions OnnxNetwork::GetOptions(int gpu, int threads,
     case OnnxProvider::TRT: {
       options.SetExecutionMode(ExecutionMode::ORT_SEQUENTIAL);
 
+      std::string cache_dir = CommandLine::BinaryDirectory() + "/trt_cache";
```

Comment:
That's not linux-friendly (where we could have e.g. `/usr/bin/lc0`). `GetUserCacheDirectory()` may be more appropriate.

Reply:
I did it explicitly like that because I'm not yet sure how portable are the engine files created. Once we know that the files are safe to keep around, then I agree `GetUserCacheDirectory()` is the right thing to use.